apps/meteor/packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -4237,9 +4237,11 @@
   "Room_archivation_state_true": "Archived",
   "Room_archived": "Room archived",
   "room_changed_announcement": "Room announcement changed to: {{room_announcement}} by {{user_by}}",
+  "Room_changed_annoucement": "changed room announcement to: {{room_announcement}}",
```

Comment:
2 things:
- There's a typo there
- Translations should always be closest as possible to the text, so this one should be `changed_room_announcement_to__room_announcement_`